worlds/pokemon_rb/__init__.py

```diff
@@ -520,6 +518,14 @@ def pre_fill(self) -> None:
                     else:
                         raise Exception("Failed to remove corresponding item while deleting unreachable Dexsanity location")
 
+        if not self.options.key_items_only:
+            loc = self.multiworld.get_location("Player's House 2F - Player's PC", self.player)
+            # Absolutely cannot have another player's item
+            if loc.item is not None and loc.item.player != self.player:
+                self.multiworld.itempool.append(loc.item)
+                loc.item = None
+            loc.place_locked_item(self.pc_item)
```

Comment:
This can place invalid items, for example, a progression item when the location is excluded.